<launch>
    <arg name="map" default="sml"/>
    <arg name="x" default="5.619764999999999"/> <!-- x offset: Mocap in the middle of the map -->
    <arg name="y" default="-5.870124000000001"/> <!-- y offset: Mocap in the middle of the map -->
    <arg name="z" default="0.0"/>
    <arg name="roll" default="0.0"/>
    <arg name="pitch" default="0.0"/>
    <arg name="yaw" default="1.57"/>
    <arg name="frame_id" default="mocap"/>
    <arg name="child_frame" default="map"/>
    <arg name="mocap_name"/>

    <include file="$(find mocap_qualisys)/launch/qualisys.launch"/>

    <node pkg="svea_mocap" type="mocap_only.py" name="mocap_svea" output="screen">
        <param name="mocap_name" value="$(arg mocap_name)"/>
    </node>

    <node pkg="svea_mocap" type="mocap_only.py" name="mocap_pedestrian" output="screen">
        <param name="mocap_name" value="pedestrian"/>
    </node>

    <include file="$(find svea_mocap)/launch/map_correction.launch">
        <arg name="map" value="$(arg map)"/>
        <arg name="x" value="$(arg x)"/>
        <arg name="y" value="$(arg y)"/>
        <arg name="z" value="$(arg z)"/>
        <arg name="yaw" value="$(arg yaw)"/>
        <arg name="roll" value="$(arg roll)"/>
        <arg name="pitch" value="$(arg pitch)"/>
        <arg name="frame_id" value="$(arg frame_id)"/>
        <arg name="child_frame" value="$(arg child_frame)"/>
    </include>

</launch>
